static/js/rules/TimeTriggerBlock.js

```diff
@@ -17,7 +17,9 @@ function TimeTriggerBlock(ruleArea, rule, x, y) {
 
   this.timeInput = document.createElement('input');
   this.timeInput.type = 'time';
-  this.timeInput.value = new Date().getHours() + ':' + new Date().getMinutes();
+  let hours = TimeTriggerBlock.leftPad(new Date().getHours());
```

Comment:
I'd suggest only getting one `Date()`. There's a small chance that these get called right at an hour transition, which could lead to a wrong time display.